es6-operators/main.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+const v1 = { value: 'Something' };
+const v2 = '';
+const v3 = undefined;
+const v4 = [3, 5, 7, 9];
+
+/* TODO:
+ * Demonstrate how to execute an expression only if a variable is truthy.
+ * In one expression, log "v1 is truthy" to the console only if `v1` is truthy.
+ * In one expression, log "v2 is truthy" to the console only if `v2` is truthy.
+ * Expected: v1 message logs, v2 message does not.
+ */
+console.log(v1 ? 'v1 is truthy' : 'v1 is falsy');
+console.log(v2 ? 'v2 is truthy' : 'v2 is falsy');
```

Comment:
Although this definitely works, for the sake of this exercise, this should be using the logical and (`&&`) instead